src/pages/Home/Home.tsx

```diff
@@ -1,19 +1,52 @@
 import React from 'react'
 import Button from '@mui/material/Button'
 import SendIcon from '@mui/icons-material/Send';
+import { useNavigate } from 'react-router-dom';
 import styles from './home.module.scss'
+import { useFetchLogoutMutation } from '../../services/AuthServices';
+import { RouterLinksName } from '../../utils/consts';
+import { useAppDispatch } from '../../hooks/redux'
+import { setCredentials, setLoginStatus } from '../../store/reducers/AuthSlice'
 
-const Home:React.FC = () => (
-  <>
-    <div className={styles.active}>Home</div>
-    <Button
-      variant='contained'
-      color='success'
-      endIcon={<SendIcon />}
-    >
-      This is my home!
-    </Button>
-  </>
-)
+const Home:React.FC = () => {
+  const dispatch = useAppDispatch()
+
+  const [fetchLogout] = useFetchLogoutMutation();
+  const navigation = useNavigate();
```

Comment:
❗️
Функции принято именовать с использованием глаголов
```suggestion
const navigate = useNavigate();
```

Reply:
Исправлено !!